test/rubocop/cop/minitest/refute_includes_test.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class RefuteIncludesTest < Minitest::Test
+  def setup
+    @cop = RuboCop::Cop::Minitest::RefuteIncludes.new
+  end
+
+  def test_registers_offense_when_using_refute_with_includes
+    assert_offense(<<~RUBY, @cop)
+      class FooTest < Minitest::Test
+        def test_do_something
+          refute(collection.includes?(actual))
```

Comment:
Please check against `include` method instead of `includes` method.
https://github.com/rubocop-hq/minitest-style-guide/pull/17

Reply:
@koic : Resolved the typo and squashed my commits. Please have a look.